packages/playground/src/utils/filter_farms.ts

```diff
@@ -0,0 +1,53 @@
+import type { InputFilterType } from "../types";
+import { isAlphanumeric, isInt, isNumeric, min, validateResourceMaxNumber } from "./validators";
+
+export type FilterFarmInputs = {
+  farmId: InputFilterType;
+  name: InputFilterType;
+  pricingPolicyId: InputFilterType;
+  totalIps: InputFilterType;
+};
+export const inputsInitializer: FilterFarmInputs = {
+  farmId: {
+    label: "Farm IDs",
+    placeholder: "e.g. 1,2,3.",
+    rules: [
+      [
+        isNumeric("Should be a number", { no_symbols: true }),
+        min("The ID should be larger then zero.", 1),
+        isInt("should be an integer"),
+      ],
+    ],
+    type: "text",
+  },
+  name: {
+    label: "Farm Name",
+    placeholder: "e.g myfarm",
+    rules: [[isAlphanumeric("Farm name should be made of either numbers or letters")]],
+    type: "text",
+  },
+  pricingPolicyId: {
+    label: "Pricing Policy",
+    placeholder: "e.g 1",
+    rules: [
+      [
+        isNumeric("This field accepts numbers only.", { no_symbols: true }),
```

Comment:
Inconsistency between error messages for the same validation error.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/101194226/4b918eee-3fe2-4f90-a3da-aef39d25968e)